src/client/Microsoft.Identity.Client/Internal/Requests/ManagedIdentityAuthRequest.cs

```diff
@@ -60,6 +60,18 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
             MsalAccessTokenCacheItem cachedAccessTokenItem = await GetCachedAccessTokenAsync()
                 .ConfigureAwait(false);
 
+            // If using IMDSv2 (mTLS) and the mTLS cert is near expiry, skip the AT cache
```

Comment:
In this file, you do not take care of requesting a POP token form the cache (for which an IAuthenticationOperation is needed)